client/src/components/App/Content/InvestigationForm/TabManagement/ClinicalDetails/ClinicalDetailsSchema.ts

```diff
@@ -74,8 +74,8 @@ const ClinicalDetailsSchema = yup.object().shape({
     [ClinicalDetailsFields.ISOLATION_ADDRESS]: yup.object().shape({
         [ClinicalDetailsFields.ISOLATION_CITY]: yup.string().required(requiredText),
         [ClinicalDetailsFields.ISOLATION_STREET]: yup.string().required(requiredText),
-        [ClinicalDetailsFields.ISOLATION_FLOOR]: yup.string().required(requiredText),
-        [ClinicalDetailsFields.ISOLATION_HOUSE_NUMBER]: yup.string().required(requiredText)
+        [ClinicalDetailsFields.ISOLATION_FLOOR]: yup.string().required(requiredText).nullable(),
+        [ClinicalDetailsFields.ISOLATION_HOUSE_NUMBER]: yup.string().required(requiredText).nullable()
```

Comment:
mixed.defined(): Schema
Mark the schema as required but nullable. All field values apart from undefined meet this requirement.